verification/curator-service/src/controllers/sources.ts

```diff
@@ -1,10 +1,13 @@
 import { Request, Response } from 'express';
 
+import { Source } from '../model/source';
+
 /**
  * List the sources.
  */
-export const list = (req: Request, res: Response): void => {
-    res.sendStatus(501);
+export const list = async (req: Request, res: Response): Promise<void> => {
+    const docs = await Source.find({});
+    res.send(docs);
     return;
```

Comment:
nit: after updating to await/async, I didn't include `return;` in my void controller methods. I don't have a preference, but would be good to standardize (I wonder if there's a linter option).

Reply:
removed the return statement as well